### ISSUE TITLE
Add rel=noopener to all target=_blank links

### ISSUE THREAD

**Issue by deg-pl** [opencaching/opencaching-pl#1154]
Link (<a>) with target=_blank without rel=noopener is a security issue. See https://mathiasbynens.github.io/rel-noopener/

**Comment by following5** [comment]
This issue is already solved by using HTMLPurifier. HTMLPurifier since version 4.8.0 automatically inserts rel="noopener" if target= is present.

**Comment by deg-pl** [comment]
(I think) we have also target=_blank links in OC PL code, not only on comments/cache desc.

**Comment by following5** [comment]
Umm... yes. If there are links to dubious websites that may want to phish OC user accounts, it may be better to completely remove those links. But inserting noopens may give a bit of extra security.

**Comment by deg-pl** [comment]
100% right! :+1:

**Comment by deg-pl** [comment]
No. Because user can publish malicious code on cache description page, user description or even cache log - with window.opener.location. We can not assume that 100% of the content published by users is safe.
